docs/examples/openiotsdk_examples.md

```diff
@@ -116,7 +135,7 @@ Then add the GDB plugin to your development environment:
     ```
 
 -   Docker container environment - mount the Fast Model Third-Party IP directory
-    into the `/opt/FastModelsPortfolio_11.16` directory in container.
+    into the `/opt/FastModelsPortfolio_11.16` directory in the container.
```

Comment:
This command is for line 149. At the moment, it is already prepopulated: https://github.com/ARM-software/connectedhomeip/blob/development/.devcontainer/devcontainer.json#L16 . 

What do you think @adbridge and @adbridge of commenting it out as the default and let people who want to debug add it like it is mentioned in these instruction ? 

I'd expect the docker image to work out of the box by default.

Reply:
Personally I think enabling for debug should be the choice rather than the default :)

Reply:
I think as people may need to change that to their own link location anyway, then commenting it out to stop errors occurring when loading the container might be the way to go..

Reply:
Let's do this in a different PR.